### PR TITLE
[IMP] open_academy: Graph, kanban and security T#59013

### PR DESCRIPTION
**Pull request by geraspace**
Hi @edy1192 I think this PR is ready to be checked. Thank you! : )

### CODE REVIEW

File: open_academy/security/security.xml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        <field name="name">Session Read</field>
+        <field name="category_id" ref="open_academy.open_academy_category"/>
+    </record>
+
+    <record id="open_academy_group_manager" model="res.groups">
+        <field name="name">Manager</field>
+        <field name="category_id" ref="open_academy.open_academy_category"/>
+    </record>
+
+    <record id="open_academy_rule_manager" model="ir.rule">
```
Comment (edy1192) [inline open_academy/security/security.xml]:
```suggestion
    <record id="open_academy_rule_responsible_can_modify" model="ir.rule">
```

File: open_academy/security/security.xml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        <field name="name">You can only modify a course if you are the responsible</field>
```
Comment (edy1192) [inline open_academy/security/security.xml]: The code must be generic, we do not name things with you, us, them...
```suggestion
        <field name="name">Only responsible can modify a course</field>
```

**Review by edy1192 — CHANGES_REQUESTED** [review]

File: open_academy/security/security.xml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        <field name="model_id" ref="model_open_academy_course"/>
+        <field name="groups" eval="[(4, ref('open_academy.open_academy_group_manager'))]"/>
+        <field name="perm_read" eval="0"/>
+        <field name="perm_write" eval="1"/>
+        <field name="perm_create" eval="0"/>
+        <field name="perm_unlink" eval="1" />
+        <field name="domain_force">
+                ['|', ('responsible_id','=',False), ('responsible_id','=',user.id)]
+        </field>
```
Comment (edy1192) [inline open_academy/security/security.xml]:
```suggestion
        <field name="domain_force">['|', ('responsible_id','=',False), ('responsible_id','=',user.id)]</field>
```

**Review by edy1192 — CHANGES_REQUESTED** [review]

File: open_academy/security/security.xml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    </record>
+
+    <record id="open_academy_rule_manager" model="ir.rule">
+        <field name="name">You can only modify a course if you are the responsible</field>
+        <field name="model_id" ref="model_open_academy_course"/>
+        <field name="groups" eval="[(4, ref('open_academy.open_academy_group_manager'))]"/>
+        <field name="perm_read" eval="0"/>
+        <field name="perm_write" eval="1"/>
+        <field name="perm_create" eval="0"/>
+        <field name="perm_unlink" eval="1" />
```
Comment (edy1192) [inline open_academy/security/security.xml]:
```suggestion
        <field name="perm_read" eval="False"/>
        <field name="perm_write" eval="True"/>
        <field name="perm_create" eval="False"/>
        <field name="perm_unlink" eval="True" />
```

**Comment by geraspace** [comment]
@edy1192 I think it's okay now : )

File: open_academy/security/security.xml [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+    <record id="open_academy_group_manager" model="res.groups">
+        <field name="name">Manager</field>
+        <field name="category_id" ref="open_academy.open_academy_category"/>
+    </record>
+
+    <record id="open_academy_rule_manager_can_modify" model="ir.rule">
+        <field name="name">Only responsible can modify a course</field>
+        <field name="model_id" ref="model_open_academy_course"/>
+        <field name="groups" eval="[(4, ref('open_academy.open_academy_group_manager'))]"/>
```
Comment (edy1192) [inline open_academy/security/security.xml]:
```suggestion
        <field name="groups" eval="[Command.link(ref('open_academy.open_academy_group_manager'))]"/>
```

**Review by edy1192 — CHANGES_REQUESTED** [review]

**Comment by geraspace** [comment]
Good morning, @edy1192. I just made the change.

**Review by edy1192 — APPROVED** [review]

**Pull request merged**